src/6model/serialization.c

```diff
@@ -10,7 +10,7 @@
 
 /* Version of the serialization format that we are currently at and lowest
  * version we support. */
-#define CURRENT_VERSION 19
+#define CURRENT_VERSION 20
```

Comment:
I  think it don't need bump it since you didn't do any version check, and looks like it is safe.